src/main/java/core/basesyntax/Lottery.java

```diff
@@ -2,13 +2,12 @@
 
 import java.util.Random;
 
-public class Lottery extends ColorSupplier {
-    public String getRundomBall() {
+public class Lottery {
+    public Ball getRandomBall() {
```

Comment:
Each time this method is called, new instances of ColorSupplier and Random are created. Let's think about how we can optimize this logic and create each instance only once.